lib/rubocop/cop/rails/find_by.rb

```diff
@@ -45,6 +46,15 @@ def autocorrect(node)
             corrector.replace(first_loc, '')
           end
         end
+
+        private
+
+        def where_with_method_as_param?(node)
+          where = node.children.find { |el| el.method?(:where) }
+          return unless where
+
+          where.arguments.size == 1 && where.arguments.first.send_type?
```

Comment:
I think this is a `Style/NumericPredicate violation. `SendNode` also implements `#first_argument`. :-)

```ruby
where.arguments.one? && where.first_argument.send_type?
```